### ISSUE TITLE
Ne pas embarquer d'aussi grosses images

### ISSUE THREAD

**Issue by pylapp** [francoismari/elyze#22]
L'application iOS fait plus de 80 Mo, ce qui est, comparé au service proposé, assez volumineux.

Le dossier _image_ fait à lui seul 16 Mo avec les photos de candidats.

Pourquoi ne pas tirer ses photographies depuis le web et les mettre en cache plutôt que de les embarquer dans l'application téléchargée des milliers, voire des millions, de fois ?

**Comment by bfontaine** [comment]
Plutôt que rajouter une couche de complexité non nécessaire, la solution simple est d’utiliser le bon format pour les images. Ce sont des photos de personnalités, donc JPEG est plus approprié que PNG (d’ailleurs certaines images en `.png` sont en réalité des jpeg). En faisant une simple conversion sans aucune optimisation, on réduit la taille totale des images à 5,6 Mo.

**Comment by WebMamba** [comment]
Je suis d'accord en optimisant les images, on peut arriver à les rendre légères. Mais la solution, la plus efficace est quand même de récupérer les images d'un serveur distant, surtout qu'on a besoin d'internet pour utiliser l'app de toute façon. Et en les récupérant d'un serveur distant, on peut les optimiser, et les mettre en cache.

**Comment by bfontaine** [comment]
Voir aussi https://github.com/francoismari/elyze/issues/23.

**Comment by francoismari** [comment]
Je note merci !
